### PR TITLE
Commerce 7018 delete post

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-7018

Added POST call for creating new PINS of SBD

### CODE REVIEW

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	setShowTooltip: PropTypes.func,
+	showTooltip: PropTypes.shape({
+		details: PropTypes.shape({
+			cx: PropTypes.double,
+			cy: PropTypes.double,
+			id: PropTypes.number,
+			label: PropTypes.string,
+			linked_to_sku: PropTypes.oneOf(['sku', 'diagram']),
+			quantity: PropTypes.number,
+
```
Comment (kresimir-coko) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js:182]: Unnecessary empty line

**Review by kresimir-coko — CHANGES_REQUESTED** [review]
Lots of formatting mistakes, please make sure to run source formatter before opening a PR.

**Comment by andrea-ale-sbarra** [comment]
ci:stop

**Comment by andrea-ale-sbarra** [comment]
ci:test:stop

**Comment by andrea-ale-sbarra** [comment]
ci:close

**Comment by andrea-ale-sbarra** [comment]
I'll will reopen this after https://github.com/liferay-frontend/liferay-portal/pull/1348 is merged

**Pull request closed without merging**